src/sections/AboutUs.jsx

```diff
@@ -0,0 +1,39 @@
+import "./about-us.css"
+
+export const AboutUs = () => {
+    const aboutArray = [
+        "/about/About1.png",
```

Comment:
Nice to put the images in an array like this. Much cleaner than import every single image. well done